core/src/main/java/fr/sncf/osrd/stdcm/preprocessing/implementation/BlockAvailabilityLegacyAdapter.java

```diff
@@ -20,7 +20,7 @@ public class BlockAvailabilityLegacyAdapter implements BlockAvailabilityInterfac
 
     /** Simple record used to group together a block and the offset of its start on the given path */
     private record BlockWithOffset(
-            Integer blockID,
+            Integer blockId,
```

Comment:
Nitpick, but this shouldn't be in the same commit

Reply:
Technically, it introduces a NM_CONFUSING bug in this commit, because I name it `blockId` in `BlockSimulationParameters`. That's the only reason why I modified this bit of code ;)